deep_beamline_simulation/tests/test_ophyd_attempt.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+import requests
+
+def test_connection():
+    session = requests.Session()
+    response_sim_list = session.post("http://localhost:8000/simulation-list", json={"simulationType": "srw"})
+    expected = '<Response [200]>'
+    assert str(response_sim_list) == expected
+
+
+def test_login():
+    session = requests.Session()
+    response_sim_list = session.post("http://localhost:8000/simulation-list", json={"simulationType": "srw"})
+    response_auth_guest_login = session.post("http://localhost:8000/auth-guest-login/srw")
+    expected = {'authState': {'avatarUrl': None,
+               'displayName': 'Guest User',
+               'guestIsOnlyMethod': True,
+               'isGuestUser': True,
+               'isLoggedIn': True,
+               'isLoginExpired': False,
+               'jobRunModeMap': {'parallel': '1 cores (SMP)',
+                                 'sequential': 'Serial'},
+               'method': 'guest',
+               'needCompleteRegistration': False,
+               'paymentPlan': 'enterprise',
+               'roles': ['adm', 'enterprise', 'premium'],
+               'uid': '78msjrcn',
+               'upgradeToPlan': None,
+               'userName': None,
+               'visibleMethods': []},
+               'state': 'ok'}
+    assert str(expected['state']) == str(response_auth_guest_login.json()['state'])
+
+
+def test_simulationUpload():
+    session = requests.Session()
+    response_sim_list = session.post("http://localhost:8000/simulation-list", json={"simulationType": "srw"})
+    response_auth_guest_login = session.post("http://localhost:8000/auth-guest-login/srw")
+    files = {"file": open("../../example-2.zip", "rb"), "folder": (None, "/foo")}
```

Comment:
use deep_beamline_simulation.__path__ to figure out the absolute path to example-2.zip

Reply:
that reminds me "example-2.zip" needs a more informative name